.github/workflows/publish.yml

```diff
@@ -1,12 +1,29 @@
 name: Publish SMB3 Foundry
 
-on:
-  push:
-    tags: [ '*' ]
+on: [push, pull_request]
 
 jobs:
-  build:
-    name: "Build"
+  code_style:
+    name: Check_Codestyle
```

Comment:
I'd prefer to keep these names as words with spaces for consistency.

Reply:
I forgot, that I can use strings there. I will fix that.